optimism/src/mips/witness.rs

```diff
@@ -616,6 +622,18 @@ impl<Fp: Field> InterpreterEnv for Env<Fp> {
         if self.preimage_bytes_read.unwrap() == preimage_len as u64 {
             let mut keccak_env = KeccakEnv::<Fp>::default();
             keccak_env.hash(self.preimage.as_ref().unwrap());
+            match self.preimage_key {
+                Some(preimage_key) => {
+                    let bytes31 = (1..32).fold(Fp::zero(), |acc, i| {
+                        acc * Fp::two_pow(8) + Fp::from(preimage_key[i])
+                    });
+                    keccak_env.add_lookup(Lookup::read_one(
```

Comment:
Typo: one has to be considered as a read, and the other one as a write. Here we write on both channels.

Reply:
We were confused, apparently there's indeed one read and one write. Maybe we were reading at a different part of the file.